native/spark-expr/src/spark_hash.rs

```diff
@@ -34,10 +34,10 @@ use datafusion::{
     error::{DataFusionError, Result},
 };
 
-use crate::execution::datafusion::expressions::xxhash64::spark_compatible_xxhash64;
+use crate::xxhash64::spark_compatible_xxhash64;
 
 #[inline]
-pub(crate) fn spark_compatible_murmur3_hash<T: AsRef<[u8]>>(data: T, seed: u32) -> u32 {
+pub fn spark_compatible_murmur3_hash<T: AsRef<[u8]>>(data: T, seed: u32) -> u32 {
```

Comment:
For consistency with other spark functions, we should probably rename this to `spark_murmur3_hash`

Reply:
done! https://github.com/apache/datafusion-comet/pull/712/commits/6780bb6e65c3d00238d087c9df533ad848b3c04f

Reply:
actually I reverted the rename since this is different form `spark_murmur3_hash` which we also have (that operates on the ColumnarValues, while this handles a single value).

Preferably this wouldn't be `pub` but it's used in the `core` crate for non-expression stuff (like shuffles) so I think it has to be.